pkg/utils/constants.go

```diff
@@ -0,0 +1,25 @@
+package utils
```

Comment:
We need the license header here: CI is complaining about it ;)

Make sure gofmt is happy too.